src/containers/RecentTrades/Market.tsx

```diff
@@ -49,91 +30,75 @@ const handleHighlightValue = (prevValue: string, curValue: string) => {
     );
 };
 
+const RecentTradesMarket = () => {
```

Comment:
please change ```Yours.tsx``` too (Not only ```Market.tsx```)

Reply:
This Pr is related just to this component